patches/src/decompression.asm

```diff
@@ -71,8 +73,8 @@ setup:
 
     ; save 16-bit register values $43xx
     REP #$20
-    LDA $08 : STA !dma_register_backup+$00
-    LDA $0A : STA !dma_register_backup+$02
+    LDA $06 : STA !dma_register_backup+$00
```

Comment:
I think register $06 ($4306) may be unsafe to use here? NMI can interrupt and do DMA transfers on channel 0, which would clobber this register. A different register that should be safe would be $54.